crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -261,6 +261,17 @@ impl EcmascriptInputTransform {
     }
 }
 
+pub fn remove_shebang(program: &mut Program) {
```

Comment:
Can we add a test case to `turbopack-tests`?

Reply:
done